src/test/java/com/googleinterns/zoomtube/utils/CommentUtilTest.java

```diff
@@ -32,12 +32,15 @@
 
 @RunWith(JUnit4.class)
 public final class CommentUtilTest {
+  private Key lectureKey;
+
   // Needed for accessing datastore services while creating an Entity.
   private final LocalServiceTestHelper testServices = new LocalServiceTestHelper();
 
   @Before
   public void setUp() {
     testServices.setUp();
+    lectureKey = KeyFactory.createKey(LectureUtil.KIND, /* id= */ 12345);
```

Comment:
I know you mentioned you made this class wide variable. This is mostly a nit just I liked having it declared in each method. You already declare parent in each method. And remember from (https://docs.google.com/document/d/18T9ef-DRqTncVD377GoIQmOGI84cNKkZ7Nn07YURFTw/edit#)

Tests tend to create many temporary values via constants, simple helper methods, or builder expressions. These values can live in class-wide constants, local variables, or be inlined directly where they are used. Inlining is often preferable for very simple values - the advantages include making its definition immediately available, reducing cognitive load by not requiring a reader to keep an additional variable in mind, and making a test read more like English by causing the object (a function argument) to follow the subject (an object on which a function is invoked).

Reply:
I also like it being declared in the test. Have switched it back